### PR TITLE
Activities 7, 8, 9 added. 

### PR DESCRIPTION
**Pull request by kimtienvu**
-Activities 7, 8, 9 added
-Took out # on main _index.md where icon image was so that it will show up
-Changed all _activity#.md to _index.md 
-Fixed Spacing issues for drag and drop blocks
-Fixed spacing issues for button and terminal placement
-Added + modified images for clarity.
-Tabs to spaces (Hopefully I got all the tabs, let me know if any are still there).

### CODE REVIEW

File: content/english/sql_workshop/activity10/_index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+{{%notice info%}}
+The Legendary Totems of Fun cannot unleash their full potential until they are merged together. They can only be merged together by the "Inventor".
+
+Your next step is to find the person who can combine both Legendary Totems of Fun. The "Inventor" will only appear if you throw a huge party!
+{{%/notice%}}
+
+## Task 10: Help the inhabitants set up for the party!
+The whole planet is invited! Help the citizens prepare for the party by finding out how many guests will be attending!
+
+Lets go over the 'SUM' Command!
```
Comment (ozhang22) [inline content/english/sql_workshop/activity10/_index.md:25]:
```suggestion
Let's go over the 'SUM' Command!
```

File: content/english/sql_workshop/activity7/_index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+title: "Use MIN Command"
+description: "Use MIN Command"
+date: 2020-07-10
+prereq: "SELECT Command, WHERE Command, MAX/MIN Command"
+difficulty: "Beginner"
+draft: false
+translationKey: "sql_activity_7"
```
Comment (ozhang22) [inline content/english/sql_workshop/activity7/_index.md:8]: FYI moving forward we are doing away with the translationKey field

```suggestion
```

File: content/english/sql_workshop/activity7/_index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+<h4 id="story"></h4>
+
+<br>
+{{< /rawhtml >}}
+
+
+### Choose and drag the correct key onto the lock to continue your adventure!
+{{%notice note%}}
+The correct key will highlight green
```
Comment (ozhang22) [inline content/english/sql_workshop/activity7/_index.md:70]:
```suggestion
The correct key will be highlighted in green
```

File: content/english/sql_workshop/activity2/_index.md [hunk 4]
```diff
@@ -29,6 +29,7 @@ The WHERE command allows you to only show data that you want.
 
 ![Ex](assets/Where_Ex.png)
 
+{{%notice info%}}
 1. The SELECT * FROM database block that we learned from the previous activity allows us to choose a database to look through.
```
Comment (Anvesha4) [inline content/english/sql_workshop/activity2/_index.md:33]: Consider styling 'select * from' and 'where' command below as code snippets

File: content/english/sql_workshop/activity5/_index.md [hunk 5]
```diff
@@ -123,9 +124,10 @@ You can use commas inside the parenthesis to choose more than one column to show
 
 </div>
 
+<div style="clear: both;"></div> 
+
 <!-- Next mission text displays -->
 <div id="text1" style="visibility:hidden">
-  <br/>
   <p> You made it! The Dear Leader tells you that the Planet of Fun is in danger of being invaded by the aliens from the Planet of Boredom!
```
Comment (Anvesha4) [inline content/english/sql_workshop/activity5/_index.md:131]: Hahaha 😂 we have to save planet of fun no matter what lol

**Pull request merged**